File: jacivan033/atividade_avaliativa1/questao13.java
import java.util.Scanner;

public class questao13 {
    public static void main (String[] args) {
        Scanner s;
        double n;
        double u;
        double i;
        double o;
        double x;
        double y;
        double d;
        s = new Scanner (System.in);
        System.out.println("Informe o x1: ");
        n = s.nextInt();
        System.out.println("Informe o x0: ");
        u = s.nextInt();
        System.out.println("Informe o y1: ");
        i = s.nextInt();
        System.out.println("Informe o y0: ");
        o = s.nextInt();
        x = (n - u) * (n - u);
        y = (o - i) * (o - i);
        d = Math.sqrt(x + y);
        System.out.println("A distancia eclidiana eh " + d);
    }
}

File: jacivan033/atividade_avaliativa1/questao13.py
print("Calculo da distancia eclidiana ")
n = float(input("Informe o x1: "))
u = float(input("Informe o x0: "))
i = float(input("Informe o y0: "))
o = float(input("Informe o y1: "))
x = (n - u) * (n - u)
y = (o - i) * (o - i)
d = (x + y) ** 0.5
print("A distancia eclidiana " + str(d))
